node/events-to-ga/src/setup.js

```diff
@@ -0,0 +1,8 @@
+import { throwIfMissing } from "./utils.js";
+
+function setup() {
+  //check for all envioromental variables
+  throwIfMissing(process.env, ["GA4_MEASUREMENT_ID", "GA4_API_SECRET"]);
+}
+
+setup();

```

Comment:
Let's check the env variables in main.js like how most of the other templates check the env vars: 

https://github.com/appwrite/templates/blob/df0b4ef612f54bbe8b3b5cf3f099137d37952ee8/node/analyze-with-perspectiveapi/src/main.js#L5

Reply:
This is still pending